webui/server.py

```
@@ -0,0 +1,195 @@
+                        "chunk": {
+                            "tools": {
+                                "messages": [{
+                                    "content": error_msg,
+                                    "status": "error"
+                                }]
+                            }
+                        }
+                    })
+                    
+    except WebSocketDisconnect:
+        print(f"WebSocket client disconnected")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)